### DISCUSSION TITLE
Alternative grid sensor is needed

### DISCUSSION THREAD

**Discussion by Coder84619** [ulic75/power-flow-card#159] [Q&A]
I have solar, and my solar combiner box has solar production values (a positive value from 0 and up), and it has consumption sensor as well for total household power usage. This value is also always positive. The amount of power going back to the grid, if any, is solar production minus power consumption. However, as I read the sensors on this card, it will not support this configuration. It wants a consumption sensor that is both positive and negative. 

I need a feature that will let me specify household grid consumption (always a positive number) and have the card do the calculation whether my net is pulling from the grid or sending back to the grid.

**Comment by kpfleming** [comment]
You can create a template sensor which combines your existing sensors, and feed the card with that.

**Comment by ulic75** [comment]
You can use [split entities](https://github.com/ulic75/power-flow-card#split-entites-example) if your battery or grid entities doesn't do both positive and negative.
